manifests/site.pp

```diff
@@ -61,3 +61,6 @@
   include role::okra
 }
 
+node 'jiraveg' {
```

Comment:
This should go away, and instead the `profile::jira` should be included in the `role::lettuce` class